### PR TITLE
ML Backend Services APIDOC

### PR DESCRIPTION
**Pull request by aks30**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @aks30,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://762.qa.docs.sunbird.org/3.9.0/ .<br/> **Note:** Only _3.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://762.qa.docs.sunbird.org/3.9.0/apis/ml-core// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-core// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-projects// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-projects// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-reports// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-reports// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-survey// <br /> - http://762.qa.docs.sunbird.org/3.9.0/apis/ml-survey// <br />  <br/> **Modified sections are -** <br />

File: apis/ml-core/ml-coreapi.yaml [hunk 1]
```diff
@@ -0,0 +1,8163 @@
+openapi: 3.0.1
+info:
+  title: Kendra Service API(s)
```
Comment (Sunitashelar) [inline apis/ml-core/ml-coreapi.yaml:3]: @aks30 @mohitga is this a DIKSHA specific API set? Should the title be 'Kendra Service'? Also, I notice that this set has a 'Create User' end point as well. How is it different from the existing Sunbird User APIs? Or is this set replacing the existing user APIs? Please clarify.

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

File: apis/ml-core/ml-coreapi.yaml (hunk 1 above)
Comment (aks30) [reply apis/ml-core/ml-coreapi.yaml:3]: Hi @Sunitashelar, we will be raising a new PR against the 4.0 release branch. Kindly ignore this for now.

**Comment by aks30** [comment]
Closing this PR for creating a new one against 4.0

**Pull request closed without merging**